api/src/main/java/org/openmrs/module/rulesengine/domain/OrderSetDrugRow.java

```diff
@@ -2,9 +2,7 @@
 
 import org.bahmni.csv.CSVEntity;
 import org.bahmni.csv.annotation.CSVHeader;
-import org.springframework.stereotype.Component;
 
-@Component
```

Comment:
how is this being used? are we autowiring this from any place? or from other dependent modules? Should there be bean def in the application context xml

Reply:
@angshu we are not auto-wiring from rulesengine/bahmni-core repo's. So removed the unnecessary annotation.